Tests/SafeDIToolTests/SafeDIToolTests.swift

```diff
@@ -1974,7 +1974,7 @@ final class SafeDIToolTests: XCTestCase {
 
             extension Root {
                 public convenience init() {
-                    func __safeDI_childABuilder(recreated: Recreated) -> ChildA {
+                    nonisolated func __safeDI_childABuilder(recreated: Recreated) -> ChildA {
```

Comment:
this part of the code-gen has me pretty convinced that `Nonisolated` was indeed the correct prefix on the `Instantiator`